product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,352 @@
+ bdr  | 5.3.0           |                   | Bi-Directional Replication for PostgreSQL
+ ```
+
+You can also confirm the other server settings using this command:
+
+```
+sudo -u enterprisedb psql edb -c "show all" | grep -e wal_level -e track_commit_timestamp -e max_worker_processes
+
+```
+
+### Create the replicated database
+
+The server is now prepared for PGD. 
+We need to next create a database named `bdrdb` and install the bdr extension when logged into it.
+
+```
+sudo -u enterprisedb psql edb -c "CREATE DATABASE bdrdb"
+sudo -u enterprisedb psql bdrdb -c "CREATE EXTENSION bdr"
+
+```
+
+Finally, test the connection by logging into the server.
+
+```
+sudo -u enterprisedb psql bdrdb
+```
+
+You should be connected to the server. 
+Execute the command "\\dx" to list extensions installed.
+
+```
+bdrdb=# \dx
+                                List of installed extensions
+       Name       | Version |   Schema   |                   Description
+------------------+---------+------------+--------------------------------------------------
+ bdr              | 5.3.0   | pg_catalog | Bi-Directional Replication for PostgreSQL
+ edb_dblink_libpq | 1.0     | pg_catalog | EnterpriseDB Foreign Data Wrapper for PostgreSQL
+ edb_dblink_oci   | 1.0     | pg_catalog | EnterpriseDB Foreign Data Wrapper for Oracle
+ edbspl           | 1.0     | pg_catalog | EDB-SPL procedural language
+ plpgsql          | 1.0     | pg_catalog | PL/pgSQL procedural language
+(5 rows)
+```
+
+Notice that the bdr extension is listed in the table, showing it is installed.
+
+## Summaries
+
+### Installing PGD for EDB Postgres Advanced Server
+
+These are all the commands used in this section gathered together for your convenience.
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+sudo systemctl start edb-as-16
+echo -e "shared_preload_libraries = '\$libdir/bdr,\$libdir/dbms_pipe,\$libdir/edb_gen,\$libdir/dbms_aq'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+echo -e "wal_level = 'logical'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+echo -e "track_commit_timestamp = 'on'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+echo -e "max_worker_processes = '16'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+sudo -u enterprisedb psql edb -c "ALTER USER enterprisedb WITH PASSWORD 'secret'"
+echo -e "host all all all md5\nhost replication all all md5" | sudo tee -a /var/lib/edb/as16/data/pg_hba.conf >/dev/null
+echo -e "*:*:*:enterprisedb:secret" | sudo -u enterprisedb tee /var/lib/edb/.pgpass >/dev/null; sudo chmod 0600 /var/lib/edb/.pgpass
+sudo systemctl restart edb-as-16
+sudo -u enterprisedb psql edb -c "CREATE DATABASE bdrdb"
+sudo -u enterprisedb psql bdrdb -c "CREATE EXTENSION bdr"
+sudo -u enterprisedb psql bdrdb
+
```

Comment:
What is the use of providing a sequence of 14 shell commands?

The users of this section are probably seeking a reference documentation which they can implement in their own processes, rather than a "quickstart" (as TPA would be quicker)

Reply:
If this was all that was on this page, you might have a point, but this page contains a reference element at the top, a worked example, and this is at the end of the page and the first of 3 summaries which go on to compare the differences between installing for EPAS, PGE and PostgreSQL.